pkgs/top-level/python-packages.nix

```diff
@@ -582,6 +582,22 @@ in modules // {
   };
 
 
+  simpleai = buildPythonPackage rec {
+    version = "0.7.11";
+    name = "simpleai";
+
+    src = pkgs.fetchurl {
+      url= "https://pypi.python.org/packages/source/s/simpleai/simpleai-${version}.tar.gz";
+      sha256 = "03frjc5jxsz9xm24jz7qa4hcp0dicgazrxkdsa2rsnir672lwkwz";
+    };
+
+    meta = {
+      homepage = https://github.com/simpleai-team/simpleai;
+      description = "This lib implements many of the artificial intelligence algorithms described on the book 'Artificial Intelligence, a Modern Approach'";
+      maintainers = with maintainers; [ NikolaMandic ];
+    };
```

Comment:
you will need to add yourself also to lib.maintainers.
https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix